subprojects/ivy/src/integTest/groovy/org/gradle/api/publish/ivy/IvyPublishJavaIntegTest.groovy

```diff
@@ -944,7 +944,6 @@ class IvyPublishJavaIntegTest extends AbstractIvyPublishIntegTest {
         run "publish"
 
         then:
-        outputContains(DefaultIvyPublication.UNSUPPORTED_FEATURE)
```

Comment:
:o: I think this should be changed to a different assertion instead of just being removed

:x: also there are several `outputDoesNotContain(DefaultIvyPublication.UNSUPPORTED_FEATURE)` in tests. I think these should all be changed to something like `outputDoesNotContain("warnings")`.